### PR TITLE
GUVNOR-3237: Change the home page to the new one

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

**Comment by paulovmr** [comment]
Part of an ensemble:
* https://github.com/kiegroup/kie-wb-common/pull/993
* https://github.com/kiegroup/drools-wb/pull/558
* https://github.com/kiegroup/jbpm-form-modeler/pull/124
* https://github.com/kiegroup/jbpm-wb/pull/814
* https://github.com/kiegroup/optaplanner-wb/pull/190
* https://github.com/kiegroup/kie-wb-distributions/pull/560

File: optaplanner-wb-webapp/src/main/resources/org/optaplanner/workbench/client/resources/i18n/AppConstants.properties [hunk 1]
```diff
@@ -8,15 +8,12 @@ AdministrationPerspective.MenuNewRepository=New Repository
...
-HomeProducer.ProjectAuthoring=Project Authoring
-HomeProducer.ArtifactRepository=Artifact Repository
-HomeProducer.Administration=Administration
-HomeProducer.DeployYourArtifacts=Deploy your Artifacts
+HomeProducer.Heading=Welcome to KIE Workbench
+HomeProducer.SubHeading=KIE Workbench offers a set of flexible tools, that support the way you need to work. Select a tool below to get started.
+HomeProducer.Design=Design
+HomeProducer.DesignDescription=Model, build, and publish your artifacts.
+HomeProducer.DevOps=DevOps
+HomeProducer.DevOpsDescription=Run and manage servers and active instances.
```
Comment (jhrcek) [inline optaplanner-wb-webapp/src/main/resources/org/optaplanner/workbench/client/resources/i18n/AppConstants.properties:16]: "active instances" of what? instances is kind of abstract and won't be enlightening for non-technical users..

**Review by mcimbora — COMMENTED** [review]
Looks like the whole menu section is missing? See  https://github.com/kiegroup/optaplanner-wb/pull/190/files#diff-904abf0c15c783a5d46cf8f8717dcb68L67

**Comment by mcimbora** [comment]
@paulovmr ^^

File: optaplanner-wb-webapp/src/main/resources/org/optaplanner/workbench/client/resources/i18n/AppConstants.properties (hunk 1 above)
Comment (paulovmr) [reply optaplanner-wb-webapp/src/main/resources/org/optaplanner/workbench/client/resources/i18n/AppConstants.properties:16]: Hi @jhrcek , could you comment that [here](https://issues.jboss.org/browse/BPMSPL-16) so the involved people can comment on that? In any case, I expect some changes on this screen in the near future, so this shouldn't be a blocker for this PR.

**Comment by paulovmr** [comment]
@mcimbora Hi, actually this is not the menu section, it is part of the old home, that contained a structure menu-like over the carousel. :)

**Comment by manstis** [comment]
@mcimbora @jhrcek Could you please re-review.

**Review by mcimbora — APPROVED** [review]

**Review by jhrcek — APPROVED** [review]

**Pull request merged**